README.md

```diff
@@ -59,6 +59,13 @@ This directive is always required.
 Maximum HTTP body size this module can generate SXG from. Default value is
 `67108864` (64 MiB).
 
+
+#### sxg\_cert\_path
+
+An absolute path in which nginx will generate and serve the CBOR-encoded certificate file.
```

Comment:
`sxg_certificate` and `sxg_certificate_key` use the phrase "full path".  Should these be the same?

Reply:
Oh, I didn't expected it. I want to distinguish them that `full path` means server internal filesystem's path, and `absolute path` means path in HTTP request.

Reply:
I understand and this makes sense from the context.